test/simulation/README.md

```diff
@@ -38,7 +38,7 @@ $ GO111MODULE=on go test -v
 - [ ] Peer disconnection should trigger orders removal to all his orders.
 
 ### Node State
-- [ ] Removed pair should trigger order removal for all the pair's orders, and removed from connected peers' order books.
+- [ ] Removed pair or swap client (lnd/raiden) disconnected (after timeout elapsed) should trigger order removal for all the pair's orders, and removed from connected peers' order books.
```

Comment:
Per discussion in #684, if we do use a timeout of ~30s before deactivating the pair that would be kind of a long delay to build into automated tests - plus we'd have to be careful it doesn't exceed the time limits for each test. What do you think @moshababo, do you know of a good way to handle a delay within a test?

Reply:
We should have the timeout length configurable, and so for the test node instance we'll do it as 1-2 sec, and it should be fine.

Reply:
I like the idea - `debug.swapClientDeactivationDelay`?

Reply:
Sounds good!

Reply:
Hm maybe make it an environment variable? If it's not something that should actually be configurable, and we'd avoid having a "debug" section in config.

Reply:
I'm good with both options. The benefit of having a debug section in the config would be automatic "documentation" when running `npm run config`. On the other hand it could be confusing for regular users.

Reply:
Right, I'd be thinking that it would be confusing for regular users. I'm thinking of this as something that we wouldn't want to have users fiddling with, only something that we want to be able to change for simulation test purposes.